app/Console/Command/ProcessDuplicatePlaces.php

```diff
@@ -203,6 +213,7 @@ private function reindexPlaces(array $placesToReIndex, OutputInterface $output,
                 $this->eventBus->publish(
                     new DomainEventStream([(new DomainMessageBuilder())->create($placeProjected)])
                 );
+                $this->duplicatePlaceRepository->markAsProcessed($placeToReIndex);
```

Comment:
@grubolsch I think there is a flaw in the new logic. The place is marked as processed but a potential error of the `UpdateLocation` is not taken into account.